src/components/Footer/Footer.tsx

```diff
@@ -30,7 +23,6 @@ export const Footer: React.FC<Props> = ({ todos, setfilterBy }) => {
           data-cy="FilterLinkAll"
           href="#/"
           className="filter__link"
-          onClick={handleClickFilter}
```

Comment:
I guess in this [comment](https://github.com/mate-academy/react_todo-app-loading-todos/pull/189#discussion_r1052419727) mentor meant you don't have to implement adding/deleting todos, etc. in this task. But the filter should work, please keep it
![image](https://user-images.githubusercontent.com/86923089/208739580-1d3fda2c-5be3-4db5-a198-0e06cc75ed97.png)